sdk/eventgrid/azure-messaging-eventgrid/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 2.0.0-beta.3 (Unreleased)
+
+
 ## 2.0.0-beta.2 (2020-09-24)
```

Comment:
I know this is already out but can we add the sectioning so that its easier to gather the changelog for release notes.
Thanks!

Reply:
Since we'll have the Oct release, will update together with the Oct release change log.